kubernetes-manifests/frontend-v2.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-v2
```

Comment:
I'd say since we're not merging this PR into master, we should fully replace the old frontend manifest with this one if possible. (It's ok for intermediate PRs to not be complete)

If that's difficult at this point, no worries though

Reply:
Sounds good, I've replaced the old frontend manifest but I'm holding off on deleting the old frontend code in this branch